Packs/AzureSentinel/Integrations/AzureSentinel/README.md

```diff
@@ -1287,3 +1286,415 @@ Create or update a watchlist item.
+| created_by | The creator of the indicator. | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| AzureSentinel.ThreatIndicator.ID | String | The ID of the indicator. | 
+| AzureSentinel.ThreatIndicator.Name | String | The name of the indicator. | 
+| AzureSentinel.ThreatIndicator.ETag | String | The ETag of the indicator. | 
+| AzureSentinel.ThreatIndicator.Type | String | The type of the indicator. | 
+| AzureSentinel.ThreatIndicator.Kind | String | The kind of the indicator. | 
+| AzureSentinel.ThreatIndicators.Confidence | Number | The confidence of the threat indicator. This is a number between 0-100. | 
+| AzureSentinel.ThreatIndicator.Created | Date | When the threat indicator was created. | 
+| AzureSentinel.ThreatIndicator.CreatedByRef | String | The creator of the indicator. | 
+| AzureSentinel.ThreatIndicator.ExternalID | String | The external ID of the indicator. | 
+| AzureSentinel.ThreatIndicator.Revoked | Boolean | Was the threat indicator revoked or not. | 
+| AzureSentinel.ThreatIndicator.Source | String | The source of the indicator. | 
+| AzureSentinel.ThreatIndicator.ETags | String | The Etags of the indicator. | 
+| AzureSentinel.ThreatIndicator.DisplayName | String | The display name of the indicator. | 
+| AzureSentinel.ThreatIndicator.Description | String | The description of the indicator. | 
+| AzureSentinel.ThreatIndicator.ThreatTypes | Unknown | The threat types of the indicator. | 
+| AzureSentinel.ThreatIndicator.KillChainPhases.KillChainName | String | The kill chain's name of the indicator. | 
+| AzureSentinel.ThreatIndicator.ParsedPattern.PatternTypeKey | Unknown | The pattern type key of the indicator. | 
+| AzureSentinel.ThreatIndicator.Pattern | String | The pattern of the indicator. | 
+| AzureSentinel.ThreatIndicator.PatternType | String | The pattern type of the indicator. | 
+| AzureSentinel.ThreatIndicator.ValidFrom | Date | The date from which the indicator is valid. | 
+| AzureSentinel.ThreatIndicator.ValidUntil | Date | The date until which the indicator is valid. | 
+| AzureSentinel.ThreatIndicator.KillChainPhases.PhaseName | String | The phase name of the indicator. | 
+| AzureSentinel.ThreatIndicator.ParsedPattern.PatternTypeValues.Value | String | The value of the indicator. | 
+| AzureSentinel.ThreatIndicator.ParsedPattern.PatternTypeValues.ValueType | String | The value type of the indicator. | 
+| AzureSentinel.ThreatIndicator.LastUpdatedTimeUtc | Date | The last updated time of the indicator. | 
+| AzureSentinel.ThreatIndicator.Tags | Unknown | The tags of the indicator. | 
+| AzureSentinel.ThreatIndicator.Types | Unknown | The threat types of the indicator. | 
+
+
+#### Command Example
+```!azure-sentinel-threat-indicator-update indicator_name=a31f2257-1af5-5eb9-bc82-acb8cc10becd display_name=WeChangedTheDisplayName indicator_type="domain-name" value=verynew.value```
```

Comment:
is this the indicator name or its id? no way to update by display name if not?

Reply:
the indicator-id is composed of it's name and more things. Each name is unique, while display-names can have duplicates.